controllers/controllers/workloads/integration/cfprocess_controller_integration_test.go

```diff
@@ -179,6 +183,33 @@ var _ = Describe("CFProcessReconciler Integration Tests", func() {
 			})
 		})
 
+		When("The process command field isn't set", func() {
+			BeforeEach(func() {
+				cfProcess.Spec.Command = ""
+			})
+
+			It("eventually creates an app workload using the newDropletCommand from the build", func() {
+				eventuallyCreatedAppWorkloadShould(testProcessGUID, testNamespace, func(g Gomega, appWorkload korifiv1alpha1.AppWorkload) {
+					var updatedCFApp korifiv1alpha1.CFApp
+					g.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(cfApp), &updatedCFApp)).To(Succeed())
+
+					g.Expect(appWorkload.OwnerReferences).To(HaveLen(1), "expected length of ownerReferences to be 1")
+					g.Expect(appWorkload.OwnerReferences[0].Name).To(Equal(cfProcess.Name))
+
+					g.Expect(appWorkload.Spec.GUID).To(Equal(cfProcess.Name))
+					g.Expect(appWorkload.Spec.Version).To(Equal(cfApp.Annotations[cfAppRevisionKey]))
+					g.Expect(appWorkload.Spec.Image).To(Equal(cfBuild.Status.Droplet.Registry.Image))
+					g.Expect(appWorkload.Spec.ImagePullSecrets).To(Equal(cfBuild.Status.Droplet.Registry.ImagePullSecrets))
+					g.Expect(appWorkload.Spec.ProcessType).To(Equal(processTypeWeb))
+					g.Expect(appWorkload.Spec.AppGUID).To(Equal(cfApp.Name))
+					g.Expect(appWorkload.Spec.Ports).To(Equal(cfProcess.Spec.Ports))
+					g.Expect(appWorkload.Spec.Instances).To(Equal(int32(*cfProcess.Spec.DesiredInstances)))
+
+					g.Expect(appWorkload.Spec.Command).To(ConsistOf("/cnb/lifecycle/launcher", newDropletCommand))
```

Comment:
I think the important bit in this `It` is to verify the workload `Spec.Command`, the rest of the checks repeat the ones on the "happy path".
I would therefore suggest that we only leave this `Exect` in this test, this would make the test easier to reason about how the context affects the result.